### DISCUSSION TITLE
Mania scroll speed depends on "HitPosition"

### DISCUSSION THREAD

**Discussion by Morilli** [ppy/osu#20818] [Ideas] [answered]
I was trying to test out the new argon skin in mania and compare it to my normal skin, and I figured out something odd: The scroll speed in lazer always feels too fast, even though I matched the scroll speed setting to the value I'm using in stable.

I figured out the cause for that behavior: The actual scroll speed changes when the `HitPosition` value in the skin.ini is changed. I guess this is somehow intentional as the scroll speed value is specified in milliseconds, and those milliseconds stay consistent, while obviously with a larger screen space the speed is higher.

I've made a simple test skin + beatmap to show the difference: 
[test mania ss.osk.zip](https://github.com/ppy/osu/files/9815413/test.mania.ss.osk.zip)
[none - -.osz.zip](https://github.com/ppy/osu/files/9815375/none.-.-.osz.zip)

Here's how the skin + beatmap looks like at scroll speed `40` in stable: 
![image](https://user-images.githubusercontent.com/35152647/196553673-81372567-94b7-46a7-8e75-9b82cda8e7f4.png)

And here in lazer: 
![image](https://user-images.githubusercontent.com/35152647/196553923-97fb69a6-b3bb-4490-8905-c4dc729b6471.png)

I feel like this should either be made more obvious or changed so the behavior is consistent with stable, as I and a friend of mine were confused by why the scroll speed felt different.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20820
